controller/scheduler/fleet.py

```diff
@@ -152,10 +152,11 @@ def _create_container(self, name, image, command, unit, **kwargs):
             f['value'] = f['value'].format(**l)
         # prepare tags only if one was provided
         tags = kwargs.get('tags', {})
-        if tags:
-            tagset = ' '.join(['"{}={}"'.format(k, v) for k, v in tags.items()])
-            unit.append({"section": "X-Fleet", "name": "MachineMetadata",
-                         "value": tagset})
+        tagset = ' '.join(['"{}={}"'.format(k, v) for k, v in tags.items()])
+        unit.append({"section": "X-Fleet", "name": "MachineMetadata",
+                     "value": tagset + ' "deisRole=general"'})
```

Comment:
worker is usualy sufficent you can add other tags via deis app if you need to be explicit